components/camera/transformpipeline/mods.go

```diff
@@ -187,16 +188,46 @@ func newCropTransform(
 	if conf.YMin >= conf.YMax {
 		return nil, camera.UnspecifiedStream, errors.New("cannot crop image to 0 height (y_min is >= y_max)")
 	}
-	cropRect := image.Rect(conf.XMin, conf.YMin, conf.XMax, conf.YMax)
+	cropRect := image.Rectangle{}
+	cropRel := []float64{}
+	if conf.XMax < 1. || conf.YMax < 1. {
```

Comment:
If I set the cropping to (0.5, 1.0) and (0.5, 1.0), I want relative cropping to the bottom right quadrant, but I'm going to get absolute cropping of the top left pixel. Rethink this logic.

Reply:
added this as a test case, and changed the logic